qa/supply-chain/imports.lock

```diff
@@ -201,12 +217,52 @@ who = "Johan Andersson <opensource@embark-studios.com>"
 criteria = "safe-to-deploy"
 version = "1.0.58"
 
+[[audits.embark-studios.audits.epaint]]
+who = "Johan Andersson <opensource@embark-studios.com>"
+criteria = "safe-to-deploy"
+violation = "<0.20.0"
+notes = "Specified crate license does not include licenses of embedded fonts if using default features or the `default_fonts` feature. Tracked in: https://github.com/emilk/egui/issues/2321"
```

Comment:
I do not see `epaint` in `Cargo.lock`, so why is this here? If we did depend on `epaint`, we'd need to include the OFL-1.0 and UFL-1.0 licenses for these fonts in `contrib/debian/copyright`.